sidecar/main.go

```diff
@@ -47,12 +47,21 @@ func main() {
 		podNamespace = flag.String("namespace", "", "namespace of the Pod that contains this sidecar")
 		podUID       = flag.String("pod-uid", "", "UID of the Pod that contains this sidecar")
```

Comment:
```suggestion
		podUID       = flag.String("pod-uid", "", "UID of the Pod that contains this sidecar")
		showVersion = flag.Bool("version", false, "Print Version details")
```

This is a bit more cleaner.

Reply:
Thanks, modified accordingly.